bazel/envoy_mobile_dependencies.bzl

```diff
@@ -22,7 +22,7 @@ def kotlin_dependencies():
         artifacts = [
             # Kotlin
             "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.11",
-
+            "androidx.recyclerview:recyclerview:1.0.0",
```

Comment:
I recommend using the latest Android X releases listed under the "stable" column here: https://developer.android.com/jetpack/androidx/versions

The 1.0.0 versions are also super (like around September 2018 or something).